saagie-zeppelin.sh

```diff
@@ -54,14 +54,14 @@ fi
 # before running Zepplin
 if [ -f "/usr/local/spark/conf/spark-env.sh" ]
 then
-  # overwrite Spark 2.1.0 config
+  # overwrite Spark config
   echo "INFO: ovewriting default spark-env.sh"
-  cp /usr/local/spark/conf/spark-env.sh /usr/local/spark/2.1.0/conf
-  chmod 755 /usr/local/spark/2.1.0/conf/spark-env.sh
+  cp /usr/local/spark/conf/spark-env.sh /usr/local/spark/${SPARK_VERSION}/conf
```

Comment:
We're using the SPARK_VERSION to find the correct PATH but Spark 2.4.5 is not installed in **/usr/local/spark/2.4.5** but in **/usr/local/spark-2.4.5-bin-hadoop2.6/**
Consequence => When running any Spark SQL command, for instance `%sql select * from default.test`
: 
_/zeppelin/bin/interpreter.sh: line 254: /usr/local/spark/2.4.5/bin/spark-submit: No such file or directory_

Reply:
corrected